### DISCUSSION TITLE
RAW file import

### DISCUSSION THREAD

**Discussion by swaechchha** [CURENT/andes#501] [Q&A] [answered]
Hi everyone,
I have imported many raw files using andes before. But recently, I have been facing an error. The file works perfectly in PSS/E but while importing in andes it gives me ''ValueError: invalid literal for int() with base 10: 'REV'''
Any insights on this would be really appreciated.

**Comment by jinningwang** [comment, marked as answer]
As discussed in documentation, ANDES supports v32 and v33 version raw file, other versions might run into error. You can try to convert the raw file to supported versions in PSSE.

https://docs.andes.app/en/latest/getting_started/formats/psse.html#raw-compatibility

**Reply by swaechchha** [reply]
Just read the documentation and found out. Thanks a lot! Just another question, importing raw file and running it in andes, the powerflow converges (also in pandapower from andes) but the load flow doesn't converge in matpower. Any idea what am I doing wrong?

**Reply by jinningwang** [reply]
It is most likely caused by power flow settings that different tools yield different convergence or results, especially when the case size is large.
1. Power flow calculators have pre-defined settings to determine the modification when reaching limits, such as active power limit, reactive power limit.
2. Different tools have different input check that might fix potential input data error.

Welcome further discussions if more detailed information is given.

**Reply by swaechchha** [reply]
Hi. I tried debugging it but can't seem to fix it. :(
The output from andes is:
![image](https://github.com/CURENT/andes/assets/111440850/b46ac7bd-f810-4f64-9c5f-3acc607762d2)
To pandapower:
![image](https://github.com/CURENT/andes/assets/111440850/82529b08-bda3-416c-b8f0-dff8db1aba32)
but for matpower:
![image](https://github.com/CURENT/andes/assets/111440850/07afb851-e1dc-409b-a278-2986cbe140c4)

I also tried changing initial guess for Vm and Va in matpower as per the output from andes but that didn't help either. Neither did changing number of iterations, convegence tolerance, etc. Really grateful for any guidance you can provide.

**Reply by jinningwang** [reply]
Make sure the MPC dict is consistent with the raw file used for ANDES.
Are you using ANDES matpower converter to get the mpc? If so, I would suggest check the converted mpc dict to see if there is any data inconsistency.
